lib/backpex/live_resource/form.ex

```diff
@@ -16,9 +16,9 @@ defmodule Backpex.LiveResource.Form do
     |> assign(:live_resource, live_resource)
     |> assign(:panels, live_resource.panels())
     |> assign(:fluid?, live_resource.config(:fluid?))
+    |> assign(:fields, live_resource.fields(live_action, socket.assigns))
```

Comment:
The fields are still in the assigns, but only to render them in the templates via `@fields` (see also `index.ex` and `show.ex`).